pgcli/main.py

```diff
@@ -160,6 +160,7 @@ def __init__(
         prompt_dsn=None,
         auto_vertical_output=False,
         warn=None,
+        show_bottom_toolbar=False,
```

Comment:
You added a parameter here, but it's not being used. The parameter would be needed if you also had a command line option to disable or enable the toolbar on-the-fly, i.e by running `pgcli --no-toolbar` (example option [here](https://github.com/dbcli/pgcli/blob/f3ac5598448a0c2080ee267f00931ed556a6779d/pgcli/main.py#L1113-L1120)), but I think that would be an overkill. Config option is enough.

Reply:
I see, need to turn on pylint, and I could try to work on that next time if I get some time. In the meantime, I have removed the unused var as per your comment.